### DISCUSSION TITLE
Use date from title as the date in the gCal code

### DISCUSSION THREAD

**Discussion by kkartch0** [YukiGasai/obsidian-google-calendar#190] [Q&A] [answered]
First off, amazing plugin! I have only used for a little bit, but I am really liking it so far. 

Second, in the README file, it mentions the ability to execute JavaScript code for calculating values such as for the date property. Would this allow for using the date from the current note's title? 

As shown below, I am wanting to display the timeline for the day that corresponds to the daily note that the timeline code is on. Any help on how I can accomplish this would be greatly appreciated!

![image](https://github.com/YukiGasai/obsidian-google-calendar/assets/46807501/05317d14-3238-455b-8f7a-af8877a92ad6)

![image](https://github.com/YukiGasai/obsidian-google-calendar/assets/46807501/a8698aac-7327-41d0-9062-eca8ee670b98)

**Comment by kkartch0** [comment]
You know, right after posting this, I had the realization of something I can do. I am still curious if there is a way to do it with JavaScript as I previously described. However, I found that it works great if I simply have date set to "{{title}}" in my daily note template, which will then be replaced with the actual date once the note is created.

**Comment by YukiGasai** [comment, marked as answer]
Hi, using the templating options is 100% the right way of doing this.
If this were an edge case where no templates or templater plugin is used, you could try:
~~~md
```gEvent
type: day
date: ;app.workspace.lastActiveFile.basename;
```
~~~

**Reply by kkartch0** [reply]
Good to know. Thank you!
